docker/repo/build.gradle

```diff
@@ -10,12 +10,12 @@ dependencies {
 project.tasks.buildDockerImage.dependsOn(":alfresco-zipkin-repo:amp")
 
 dockerAlfresco {
-    baseImage = "xeniteu/alfresco-repository-community:5.2.g"
+    baseImage = "xenit/alfresco-repository-community:5.2.g"
```

Comment:
Can you prefix the images from hub.docker.com with `docker.io` please ?

Reply:
ah yes, good point